src/app/utils/testing.utils.ts

```diff
@@ -217,3 +228,155 @@ function mockPlates(args?: Partial<Plate>): Plate {
   };
   return { ...mock, ...args };
 }
+
+function mockAuthoIntoService(args?: Partial<AuthoIntoService>): AuthoIntoService {
+  const mock: AuthoIntoService = {
+    cocIssueDate: '2019-12-13',
+    dateReceived: '2018-11-13',
+    dateAuthorised: '2018-2-10',
+    datePending: '2017-6-5',
+    dateRejected: ''
+  };
+  return { ...mock, ...args };
+}
+
+function mockLettersOfAuth(args?: Partial<LettersOfAuth>): LettersOfAuth {
+  const mock: LettersOfAuth = {
+    letterType: 'Trailer authorization',
+    letterDateRequested: '2020-1-13',
+    letterContents: 'some letter text'
+  };
+  return { ...mock, ...args };
+}
+
+function mockPurchaser(args?: Partial<PurchaserDetails>): PurchaserDetails {
+  const mock: PurchaserDetails = {
+    name: 'Ben',
+    address1: 'Garrets green',
+    address2: 'Birmingham',
+    address3: 'lala land',
+    postCode: 'NG4 12Z',
+    postTown: 'A town',
+    telephoneNumber: '123123123',
+    emailAddress: 'test@test.com',
+    faxNumber: '1231234',
+    purchaserNotes: 'some notes'
+  };
+  return { ...mock, ...args };
+}
+
+function mockManufacturer(args?: Partial<ManufacturerDetails>): ManufacturerDetails {
+  const mock: ManufacturerDetails = {
+    name: 'Ben',
+    address1: 'Garrets green',
+    address2: 'Birmingham',
+    address3: 'lala land',
+    postCode: 'NG4 12Z',
+    postTown: 'A town',
+    telephoneNumber: '123123123',
+    emailAddress: 'test@test.com',
+    faxNumber: '1231234',
+    manufacturerNotes: 'some notes'
+  };
+  return { ...mock, ...args };
+}
+
+
+function mockTechRecHistory(args?: Partial<VehicleTechRecordModel>): VehicleTechRecordModel {
+  const tecRec: TechRecord = {
+                              statusCode: 'current',
+                              reasonForCreation: 'reason for creation',
+                              createdByName: 'test name',
+                              createdAt: '1-1-2020',
+                              lastUpdatedByName: 'test 1',
+                              lastUpdatedAt: '12-3-2020'
+                             } as TechRecord;
+  const mock: VehicleTechRecordModel = {techRecord: [tecRec]} as VehicleTechRecordModel;
+  return { ...mock, ...args };
+}
+
+function mockVehicleSummary(args?: Partial<TechRecord>): TechRecord {
```

Comment:
we wont need this mockVehicleSummary. mockTechRecord is sufficient

Reply:
updated as needed